### DISCUSSION TITLE
Confidence interval for c-index on external test set using subsample?

### DISCUSSION THREAD

**Discussion by jaylusk1** [kogalur/randomForestSRC#425] [Q&A]
I have successfully followed the steps in the following discussion post (https://github.com/kogalur/randomForestSRC/discussions/134) to calculate the 95% confidence interval for the c-index of my model using subsample. However, I am trying to calculate the 95% CI for the model when applied to an external test set, and the subsample function will not take an input class of rfsrc, predict. Does anyone know how I can modify my predict object or the subsample code to calculate the 95% CI for the c-index on an external test set?

**Comment by ishwaran** [comment]
Hi, the `subsample` function only accepts objects which are grow objects (i.e. the training forest).  We will consider adding the ability to construct confidence intervals for the learner applied to test data in future releases.
